single_kernel_mongo/workload/__init__.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""The different workloads and their code for mongo charms."""
```

Comment:
in our 1:1 I would love an explanation on this file :)

Reply:
Sure (and beware, I don't like the solution here but haven't found time to think about and write something way better.)